frontend/src/app/shared/components/modal/modal-overlay.sass

```diff
@@ -9,7 +9,6 @@
   justify-content: center
   align-items: center
   display: none
-  pointer-events: none
 
   > *
```

Comment:
You can also remove this

Reply:
Yes I have to do it, it is pushed by mistake